crates/cxx-qt-lib-extras/src/assertion_utils.h

```diff
@@ -3,52 +3,9 @@
 // clang-format on
```

Comment:
can this file just be removed? as now `#include <cxx-qt-lib/assertion_utils.h>` is used?